examples/Hermitian_Phasing.py

```diff
@@ -67,10 +52,13 @@
         fl.next("Zeroth order phasing correction applied")
         fl.image(data)
         #{{{ Applying the phase corrections
+        fl.basename = "(Hermitian diagnostic)"
         best_shift, max_shift = hermitian_function_test(
             select_pathway(data.C.mean(indirect), signal_pathway),fl=fl)
+        fl.basename = None
         data.setaxis("t2", lambda x: x - best_shift).register_axis({"t2": 0})
-        fl.next('After Hermitian Test')
+        fl.next('After Hermitian Test, time domain')
+        fl.image(data)
+        data.ft("t2")
+        fl.next('After Hermitian Test, frequency domain')
         fl.image(data)
```

Comment:
added in frequency domain, as well -- so what shows up actually does work, so the issue might be your implementation elsewhere

Reply:
I think you are right. So is this example necessary if it is shown in the correlation alignment example?

Reply:
yes -- for two reasons:

- in case it seems like it's doing something funky, and we want to check (the reason we're using it now)
- in case someone *just* wants to phase their data, without correlation, etc -- which is a lot of the time, actually